pulse/urls/question_urls.py

```diff
@@ -1,10 +1,12 @@
 # urls/question_urls.py
 from django.urls import path
-from ..views import question_views
+from pulse.views.question_views import getAllQuestions
+from pulse.views import question_views
 
 # URL routes for calls relating to questions
 urlpatterns = [
     path('create/', question_views.createQuestion, name='createQuestion'),
     path('getAll/', question_views.getAllQuestions, name='getAllQuestions'),
     path('getById/<str:question_id>/', question_views.getQuestionById, name='getQuestionById'),
+    path('api/getAllQuestions/', getAllQuestions, name='get_all_questions')
```

Comment:
I think this endpoint is already handled by line 9 (i.e. `geAll`). I would remove line 11.

Also, generally it's not needed to include `api` as a prefix for the endpoint